docs/reference/ingest/processors/geoip.asciidoc

```diff
@@ -59,6 +59,8 @@ in `properties`.
 * If the GeoIP2 Anonymous IP database is used, then the following fields may be added under the `target_field`: `ip`,
 `hosting_provider`, `tor_exit_node`, `anonymous_vpn`, `anonymous`, `public_proxy`, and `residential_proxy`. The fields actually added
 depend on what has been found and which properties were configured in `properties`.
+* If the GeoIP2 Connection Type database is used, then the following fields may be added under the `target_field`: `ip`, and
+`connection_type`. The fields actually added depend on what has been found and which properties were configured in `properties`.
```

Comment:
We need to also mention `connection_type` in the enterprise database section.

Reply:
1523e1fd6af0faab1ac14f0b2d90b609d9f5d925